sprints/dashboard/models.py

```diff
@@ -140,9 +164,10 @@ def review_time(self) -> int:
             if self.is_epic or self.status in settings.SPRINT_STATUS_NO_MORE_REVIEW:
                 return 0
 
-            if self.story_points <= 3:
-                return SECONDS_IN_HOUR
-            return int(2 * SECONDS_IN_HOUR)
+            review_time = self.get_review_time_from_story_points()
+            if review_time is None:  # if some value is not defined on the `SPRINT_HOURS_RESERVED_FOR_REVIEW` setting
+                raise ValueError
```

Comment:
Is this exception handled? If not, it's going to break the dashboard. See my previous comment - we probably don't need this.

Reply:
Removed the exception